### PR TITLE
Add Team Ross to Sprint Teams list

### PR DESCRIPTION
**Pull request by holytoastr**
### Change notes:
- Added Team Ross to the sprint teams yml file
- Updated gems that were preventing local testing

### Screenshots
Screen Shot 2022-07-13 at 10 56 23 AM
Screen Shot 2022-07-13 at 12 16 16 PM

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,116 +1,113 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (6.0.3.2)
+    activesupport (7.0.3.1)
```
Comment (zachmargolis) [inline Gemfile.lock:4]: should we update/pin these in the `Gemfile` as well?

File: _data/sprint_teams.yml [hunk 2]
```diff
@@ -60,6 +60,12 @@
   slack_channel_url: https://gsa-tts.slack.com/archives/C01DBBNTL68
   slack_channel_name: login-team-radia
 
+- name: Ross
+  namesake_markdown: "[Mary G Ross](https://en.wikipedia.org/wiki/Mary_Golda_Ross), the first known Native American female engineer, she worked on the preliminary design concepts for interplanetary space travel, manned space flight, ballistic missiles, and satillites in orbit above the earth"
```
Comment (zachmargolis) [inline _data/sprint_teams.yml:64]: small typo

```suggestion
  namesake_markdown: "[Mary G Ross](https://en.wikipedia.org/wiki/Mary_Golda_Ross), the first known Native American female engineer, she worked on the preliminary design concepts for interplanetary space travel, manned space flight, ballistic missiles, and satellites in orbit above the earth"
```

**Review by zachmargolis — APPROVED** [review]
LGTM

File: Gemfile.lock (hunk 1 above)
Comment (holytoastr) [reply Gemfile.lock:4]: The `Gemfile` didn't specify a specific version, but I am happy to add that

**Pull request merged**